app/AssessmentItem.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use App\Assessment;
+
+class AssessmentItem extends Model
+{
+    public function assessment() {
```

Comment:
you should try and set your IDE to use PSR-1 and PSR-2 formatting.

when you reformat the file, it will follow this widely used convention so you don't have to think about it.

you can get phpstorm to reformat on save as well.

ie. 

```
public function assessment() 
{
...
}
```
is the correct formatting.